File: code/examples/sample.py
"""
Sample Python Test Code

This is a simple Python example demonstrating basic testing concepts.
"""

def add(a, b):
    """Add two numbers and return the result."""
    return a + b

def subtract(a, b):
    """Subtract b from a and return the result."""
    return a - b

def multiply(a, b):
    """Multiply two numbers and return the result."""
    return a * b

def divide(a, b):
    """Divide a by b and return the result."""
    if b == 0:
        raise ValueError("Cannot divide by zero")
    return a / b

# Example usage
if __name__ == "__main__":
    print("Addition: 5 + 3 =", add(5, 3))
    print("Subtraction: 5 - 3 =", subtract(5, 3))
    print("Multiplication: 5 * 3 =", multiply(5, 3))
    print("Division: 6 / 3 =", divide(6, 3))

File: code/examples/Sample.java
/**
 * Sample Java Test Code
 * 
 * This is a simple Java example demonstrating basic testing concepts.
 */

public class Sample {
    
    /**
     * Add two numbers
     */
    public static int add(int a, int b) {
        return a + b;
    }
    
    /**
     * Subtract b from a
     */
    public static int subtract(int a, int b) {
        return a - b;
    }
    
    /**
     * Multiply two numbers
     */
    public static int multiply(int a, int b) {
        return a * b;
    }
    
    /**
     * Divide a by b
     */
    public static double divide(int a, int b) {
        if (b == 0) {
            throw new IllegalArgumentException("Cannot divide by zero");
        }
        return (double) a / b;
    }
    
    /**
     * Main method for testing
     */
    public static void main(String[] args) {
        System.out.println("Addition: 5 + 3 = " + add(5, 3));
        System.out.println("Subtraction: 5 - 3 = " + subtract(5, 3));
        System.out.println("Multiplication: 5 * 3 = " + multiply(5, 3));
        System.out.println("Division: 6 / 3 = " + divide(6, 3));
    }
}
